nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-core/src/test/java/org/nuxeo/ecm/directory/memory/TestMemoryDirectory.java

```diff
@@ -479,7 +481,7 @@ public void testGetProjection() throws Exception {
 
     // actually tests AbstractDirectory.orderEntry
     @Test
-    public void testOrderBy() throws Exception {
+    public void testOrderBy() {
         Map<String, Object> e2 = new HashMap<>();
```

Comment:
wdyt if we replace the HashMap by `Map.of`, as already you use `Set.of`.

Reply:
Indeed, I think we should. It's just that I changed a lot of files and I didn't already do the cleanup step for now.